src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_search_objects.py

```diff
@@ -14,10 +14,10 @@
 def create_industrial_search_indexed_objects() -> None:
     if settings.ALGOLIA_TRIGGER_INDEXATION:
         logger.info("create_industrial_algolia_objects")
-        offer_ids = Offer.query.options(orm.load_only(Offer.id)).all()
+        offer_ids = [offer.id for offer in Offer.query.options(orm.load_only(Offer.id))]
```

Comment:
Ça marche. Ou sinon :

```suggestion
        offer_ids = [offer_id for offer_id, in Offer.query.with_entities(Offer.id)]
```

... qui évite de créer un objet Offer.